src/components/ErrorModal/ErrorModal.js

```diff
@@ -38,20 +38,19 @@ class ErrorModal extends React.PureComponent {
   onError = error => {
     const { openElement, documentPath, t } = this.props;
```

Comment:
Had to change the patch to this function because changes broke it.  It hides the 'network error' message so book-admin should probably pay particular attention to it (I think director page-caching.service never tries to load invalid caches anymore).